draft-kowalik-rpp-architecture-01.adoc

```diff
@@ -243,14 +243,16 @@ Other transform operations like renew, or restore which are not addressable reso
 This basic set of rules and guidelines will be further refined in the RPP specifications and give an universal toolset for extending RPP with new resources and commands.
 
 
-==== RPP specific error codes and relation to HTTP error codes
+==== RPP specific status codes and relation to HTTP status codes
 
-RPP utilizes both HTTP status codes and RPP-specific error codes within RPP-specific HTTP Headers and the response body for detailed error reporting and allowing the client or an intermediate to determine what action to take based on status code and header details only.
+RPP utilizes both HTTP status codes and RPP-specific status codes, using RPP-specific HTTP Headers. The response body may contain machine readable problem details <<RFC7807>> for detailed status reporting. Using HTTP status codes and headers allows the client or an intermediate (HTTP status only) to determine what action to take based on status code and header details only.

```

Comment:
Problem details can be used really only for errors, as it has own representation so it can be hardly used for warning or success information.

For info/warning, if we don't want to add it to representation, it should go into headers as well.